third_party/3/jwt/contrib/algorithms/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Callable, Any, Union
+from hashlib import _DataType
+
+# In reality, _HashAlg is a function of the type
+# that hashlib.sha256/384/512 are, but there doesn't
+# seem to be a consistent exportable name that we can reference
+# for that, so we'll just say it's a Callable here.
```

Comment:
Hmm, so this is what I'd tried originally, but I get the error `Module 'hashlib' has no attribute '_Hash'`.

I noticed that in the Python 2 stubs, this type is called `_hash`, so I also tried importing that name--just in case my mypy is somehow using the python 2 stubs--but that doesn't work either.

Any ideas?